export_tsvs.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import hail as hl
+import os
+
+p = argparse.ArgumentParser()
+p.add_argument("-o", "--output-bucket-path", help="Path where the .tsvs for each sample will be written.", default="gs://gnomad-bw2/gnomad_readviz_tsvs")
+p.add_argument("sample_ids_file_path", help="A text file containing one sample id per line")
+args = p.parse_args()
+
+# parse sample ids
```

Comment:
Could you expand on this comment and the choice of i == 20? maybe that could be an import param?

Reply:
This is just a logging detail, so doesn't seem worth a separate parameter. The linux `head` command prints 10 items by default if you run it without -n, and here I just picked 20. I'll change it to 10.